packages/clerk-js/src/ui/common/withRedirectToHome.tsx

```diff
@@ -16,7 +16,7 @@ export function withRedirectToHome<P extends SignInProps | SignUpProps>(
     const { singleSessionMode } = authConfig;
     const session = useCoreSession({ avoidUndefinedCheck: true });
 
-    React.useEffect(() => {
+    useEffect(() => {
```

Comment:
This is an example of a file throwing an error in tests. Swapping the imports didn't help

Reply:
@dvargas92495 I am not sure I get what the problem was with tests and this change. Can you please elaborate?

Reply:
looking at the CI failure, I'm now pretty sure it's due to the mismatched React versions between tests and now clerk-js using React 18:

![image](https://user-images.githubusercontent.com/7143571/167155659-5a76d2ba-9542-4ff8-be0d-22a3ea6337ac.png)

As I mentioned in the comment below, I think I know how to iterate from here and am unblocked for now

Reply:
Yes, that should be it. Let's make sure that only React 18 is used.